README.md

```diff
@@ -95,10 +95,38 @@ needed:
 
 ### Testing Changes Locally
 
-#### Configure Your Environment
+There are two ways to test changes to the `docs/` directory locally: via Docker
+or getting all the dependencies installed on your system.
 
-To test changes to the `docs/` directory locally, follow these steps to make
-sure you have the GitHub Pages dependencies installed:
+#### Via docker
+
+##### Configure Your Environment
+
+Firstly, make sure that you have [Docker](https://www.docker.com/) installed.
+Then from a command prompt, type the following commands:
+
+``` bash
+git clone https://github.com/phpcommunity/phptestfest.org.git
+cd phptestfest.org/docs/
+```
+
+##### Previewing Changes
+
+To preview your changes, type the following into a command prompt from the
+`docs/` directory:
+
+``` bash
+docker run --rm -it -p 4000:4000 -v "$PWD":/app -w /app etelej/jekyll
```

Comment:
Change to `docker run -v "$PWD":/usr/src/app -p "4000:4000" phpcommunity/github-pages`